articles/extensions/authorization-extension/v2/implementation/configuration.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Configuring the Authorization Extension
+description: How to configure the Authorization Extension
+toc: true
+---
+
+# Authorization Extension: Configuration
+
+Before the extension can enforce your authorization logic, you'll need to configure how it will behave during the login transaction. Your configuration settings will be captured in a [rule](/rules) that's executed during runtime.
+
+## Configure the Extension
+
+Open up the Authorization Extension, and click **Configuration** on the drop-down menu in the top right of the **Authorization Dashboard**.
+
+![Click Configuration](/media/articles/extensions/authorization/click-configuration.png)
+
+This brings you to the **Rule Configuration** section of the **Configuration** page.
+
+![Configuration page](/media/articles/extensions/authorization/configuration.png)
+
+All of the changes you make in the sections under **Token Contents**, such as those related to groups, roles, and permissions, will be reflected in the rule you export at the completion of this step. 
+
+### Data Limitations
```

Comment:
This looks more like a warning panel that should be placed somewhere else (this is the first time you refer to a token and we talk directly about limitations, it's a bit confusing). Maybe move it at the end of the next paragraph?

Reply:
Also, the persistence link is broken, I assume it should be #store-groups-roles-and-permissions-information-in-the-users-profiles ?